### ISSUE TITLE
Formula to calculate `survival_time_in_days` when patient is alive

### ISSUE THREAD

**Issue by sujaypatil96** [monarch-initiative/oncoexporter#87]
Notes from discussion with @justaddcoffee on 4/16.

If person/patient is "Alive" then we need a way of populating survival time in days, and we can do that as follows:

```
when we last saw the patient (day of last encounter) - when the patient was diagnosed
```

In the [GDC API](https://docs.gdc.cancer.gov/API/Users_Guide/Appendix_A_Available_Fields/), the field that corresponds to each of the above variables is:
* when we last saw the patient (day of last encounter) -- diagnoses.days_to_last_follow_up
* when the patient was diagnosed -- diagnoses.age_at_diagnosis

**Comment by justaddcoffee** [comment]
@sujaypatil96 we hacked on this a bit today. We observed that in the current code, survival time for Alive patients was often negative. 

We think that is because last_follow_up means days since diagnosis (not days since birth):
Via: https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=diagnosis&anchor=days_to_last_follow_up
"Time interval from the date of last follow up to the date of initial pathologic diagnosis, represented as a calculated number of days."

So, we've changed the code accordingly. Does this look correct to you?

**Comment by sujaypatil96** [comment]
@justaddcoffee yup, the code makes sense to me. All that needs to be returned is the "days_to_last_follow_up".

Do we need to comment out the code that makes calls to [_process_vital_status()](https://github.com/monarch-initiative/oncoexporter/blob/develop/src/oncoexporter/cda/cda_individual_factory.py#L36-L67) here?
* https://github.com/monarch-initiative/oncoexporter/blob/develop/src/oncoexporter/cda/cda_individual_factory.py#L98C26-L98C47
* https://github.com/monarch-initiative/oncoexporter/blob/develop/src/oncoexporter/cda/cda_individual_factory.py#L113-L114

**Comment by justaddcoffee** [comment]
@sujaypatil96 is [_process_vital_status()](https://github.com/monarch-initiative/oncoexporter/blob/develop/src/oncoexporter/cda/cda_individual_factory.py#L36-L67) not being used? I just ran the code through a debugger and it seems this is still the way we are populating vital status for the phenopacket, right?
